### PR TITLE
osc.lua: fix refactor breaking user configurable colors

### PR DESCRIPTION
**Pull request by UNIVPM-S1079707**

### CODE REVIEW

File: player/lua/osc.lua [hunk 1]
```diff
@@ -106,28 +106,53 @@ end
...
+
+    topButtonsBar = "",
+    smallButtonsBar = "",
+    timecodesBar = "",
+    timePosBar = "",
+    vidtitleBar = "",
+
+    wcButtons = "",
+    wcTitle = "",
+    wcBar = "",
```
Comment (kasper93) [inline player/lua/osc.lua:128]: Just keep it nil untill it is set.
```
local osc_styles
```
No need for default init of each value.

File: player/lua/osc.lua [hunk 2]
```diff
@@ -2978,6 +2982,7 @@ end)
 
 validate_user_opts()
 visibility_mode(user_opts.visibility, true)
+set_osc_styles()
```
Comment (kasper93) [inline player/lua/osc.lua:2985]: put it also in the `read_options` callback for runtime updates.

Comment (UNIVPM-S1079707) [reply player/lua/osc.lua:2986]: Inside `opt.readoptions()`? (pushed)

Comment (kasper93) [reply player/lua/osc.lua:2986]: Yes, and like I said, put it immediately after `validate_user_opts`. Important to do it before `request_tick`.

File: player/lua/osc.lua [hunk 3]
```diff
@@ -1,4 +1,4 @@
-local assdraw = require 'mp.assdraw'
+local assdraw
```
Comment (kasper93) [inline player/lua/osc.lua:1]: unwanted change?

Comment (UNIVPM-S1079707) [reply player/lua/osc.lua]: Yeah, LSP stufff. Fixed.

**Review by kasper93 — APPROVED** [review]

**Pull request merged**